crowdsec-docs/unversioned/user_guides/waf_rp_howto.md

```diff
@@ -0,0 +1,277 @@
+
+We do have our Nginx logs:
+
+```bash
+==> /var/log/nginx/access.log <==
+A.B.C.D - - [22/May/2025:08:32:49 +0000] "GET / HTTP/1.1" 200 3121 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/136.0.0.0 Safari/537.36"
+A.B.C.D - - [22/May/2025:08:32:49 +0000] "GET /icons/ubuntu-logo.png HTTP/1.1" 200 3322 "http://3.252.135.216/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/136.0.0.0 Safari/537.36"
+A.B.C.D - - [22/May/2025:08:32:50 +0000] "GET /favicon.ico HTTP/1.1" 404 245 "http://3.252.135.216/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/136.0.0.0 Safari/537.36"
+```
+
+And the Apache logs:
+
+```bash
+==> /var/log/apache2/access.log <==
+A.B.C.D - - [22/May/2025:08:32:49 +0000] "GET / HTTP/1.1" 200 3404 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/136.0.0.0 Safari/537.36"
+A.B.C.D - - [22/May/2025:08:32:49 +0000] "GET /icons/ubuntu-logo.png HTTP/1.1" 200 3552 "http://3.252.135.216/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/136.0.0.0 Safari/537.36"
+A.B.C.D - - [22/May/2025:08:32:50 +0000] "GET /favicon.ico HTTP/1.1" 404 438 "http://3.252.135.216/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/136.0.0.0 Safari/537.36"
+
+
+```
+
+:::warning
+
+At this stage, check that the IPs appearing in your Apache's and Nginx's log are the real originating IP.
+
+:::
+
+## Time to beef up our security - Security Engine
+
+As soon as our server is online, hordes of malevolent IPs will jump on it with clear bad intentions. What is currently happening is this:
+
+![initial setup](/img/original-setup.png)
+
+Thus, it is time to step up our security with CrowdSec. We will deploy the Security Engine, the WAF, and an Nginx bouncer on our Reverse Proxy server so that we can achieve this:
+
+![harden setup](/img/harden-setup.png)
+
+To install CrowdSec on our reverse proxy, let’s grab the crowdsec repository:
+
+```bash
+$ curl -s https://install.crowdsec.net | sudo sh
+```
+
+And let’s install crowdsec:
+
+```bash
+# apt install crowdsec
+```
+
+The relevant part of the install log is the following:
+
+```bash
+Creating /etc/crowdsec/acquis.yaml
+INFO[2025-05-22 09:40:27] crowdsec_wizard: service 'nginx': /var/log/nginx/error.log /var/log/nginx/access.log
+INFO[2025-05-22 09:40:27] crowdsec_wizard: service 'ssh': /var/log/auth.log
+INFO[2025-05-22 09:40:27] crowdsec_wizard: service 'linux': /var/log/syslog /var/log/kern.log
+```
+
+It detected we’re running nginx, and will automatically install the relevant scenarios and start monitoring the associated logfiles!
+
+Then, we will enrol the SE in the console:
+
+![cscli enroll](/img/cscli-enroll.png)
+
+Accept it in the console:
+
+![enroll console](/img/cscli-enroll-console-view.png)
+
+## Detecting is cool, blocking is better.
```

Comment:
"detecting is great but blocking is better" feel free to remove the `but` for `,`